packages/react-menu/src/components/MenuList/useMenuList.ts

```diff
@@ -100,3 +109,34 @@ export const useMenuList = (
 
   return state;
 };
+
+/**
+ * Adds some sugar to fetching multiple context selector values
+ */
+const useMenuContextSelectors = () => {
+  const hasMenuContext = useMenuContext(context => context.hasMenuContext);
+  const checkedValues = useMenuContext(context => context.checkedValues);
```

Comment:
_Wondering, do we need `context-selector` there?_

We need to pass `checkedValues` down and they will be unstable in 99% so we will not improve actually anything...

Reply:
You're right, there's no sense in it, it is actually equivalent to receiving all those values as props

Reply:
On second thought, I will keep context selector because the `MenuTrigger` will use `setOpen` callback to toggle popup